dev-libs/mimetic/mimetic-0.9.8.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="Email library (MIME) written in C++"
+DESCRIPTION="A powerful, full featured, STL-based, standards compilant and \
+free C++ MIME library."
+HOMEPAGE="http://www.codesink.org/mimetic_mime_library.html"
+COPYRIGHT="2003-2014 Stefano Barbato"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="http://www.codesink.org/download/mimetic-$portVersion.tar.gz"
+CHECKSUM_SHA256="3a07d68d125f5e132949b078c7275d5eb0078dd649079bd510dd12b969096700"
+PATCHES="mimetic-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
+
+PROVIDES="
+	libmimetic$secondaryArchSuffix = $portVersion
+	lib:libmimetic$secondaryArchSuffix = 0.0.0
```

Comment:
I don't know if this is required, but it looks like we always add a compat.
So maybe you could add a "compat >= 0" here too?